backoffice/common/services/ResponseStatusHandlingService.ts

```diff
@@ -51,3 +51,15 @@ export const handleCreatingResponse = async (response: any) => {
     toastError(CREATE_FAILED);
   }
 };
+
+export const handleResponse = (response: any, successMsg: string | any, errorMsg: string | any) => {
+  if (response.status === ResponseStatus.SUCCESS) {
+    toastSuccess(successMsg);
+  } else if (response.status === ResponseStatus.NOT_FOUND) {
+    toastError(errorMsg);
+  } else if (response.status === ResponseStatus.BAD_REQUEST) {
+    toastError(errorMsg);
+  } else {
+    toastError(errorMsg);
```

Comment:
Consider using a switch case like this:
switch(response.status) {
 case ResponseStatus.SUCCESS:
   toastSuccess(successMsg);
   break;
 case ResponseStatus.NOT_FOUND:
 case ResponseStatus.BAD_REQUEST:
   toastError(errorMsg);
   break;
 default:
   break;
}